src/run-codemod-on-file.ts

```diff
@@ -138,20 +138,21 @@ export default async function runCodemodOnFile(
       parse(source: string, opts: Record<string, unknown>) {
         const babelOpts = {
           ...getBabelOpts(),
-          ..._.pick(codemod, 'presets'),
+          ..._.pick(codemod, 'babelTransformOptions'),
```

Comment:
This isn't quite what we want. `_.pick(a, 'b')` results in `{b}`, so our spread is:

```js
{
  ...getBabelOpts(),
  ...{babelTransformOptions}  
}
```

We just want `...codemod.babelTransformOptions`.

I made this fix on my local branch.

Reply:
Yeah my lodash is rusty. I don't understand why `_.pick` is needed here at all, vs just:

```ts
...getBabelOpts(),
presets: codemod.presets,
```

But I was trying to align with conventions of the project.